syft/frameworks/torch/tensors/interpreters/multi_pointer.py

```diff
@@ -103,7 +103,6 @@ def shape(self) -> torch.Size:
     def dim(self) -> int:
         """This method fixes the error that the result of dim was a list of ints
         stored inside a multipointer tensor"""
-
```

Comment:
Can you undo this change? Just so we don't track in unedeed file.

Reply:
Should be done now